client/package.json

```diff
@@ -187,7 +186,7 @@
     "test-jest": "export ANET_CONFIG=../anet.yml ; NODE_ENV=test jest --testPathPattern='tests/(actions|reducers|models)'",
     "test-wdio-ie": "export SERVER_URL=${SERVER_URL:-'http://localhost:8180'} ; NODE_ENV=test node ./tests/util/wdio.ie.js",
     "build-storybook": "build-storybook",
-    "storybook": "start-storybook -p 6006",
+    "storybook": "export STORYBOOK_ANET_CONFIG=/anet.yml ; start-storybook -s .. -p 6006",
```

Comment:
In other places, we export anet config like this `CONFIG=../anet.yml`, why is this different?

Reply:
It's an HTTP request from the Storybook server (can't access a file directly, as it all runs in the browser). So we just map the top-level ANET directory as Storybook's server's root for serving static files, and request `/anet.yml` from that static area.